packages/plugin/src/theia.d.ts

```diff
@@ -2770,6 +2770,11 @@ export module '@theia/plugin' {
          */
         readonly processId: PromiseLike<number>;
 
+        /**
+         * The exit status of the terminal, this will be undefined while the terminal is active.
+         */
+        readonly exitStatus: TerminalExitStatus | undefined;
```

Comment:
VScode's API leaves a complete example in comment for this field. I wasn't sure it was a good idea to have it in theia's API, so I removed it.

Reply:
Yes, it's still valid for Theia. I will include it in the next commit.